packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -169,8 +134,8 @@ export class EditorStore implements CommandRegistrar {
   sdlcState: EditorSDLCState;
   graphState: EditorGraphState;
   graphManagerState: GraphManagerState;
```

Comment:
Hm, in terms of organization at `EditorStore` level, I think we should move `EditorStore.graphEditorMode` to `EditorStore.graphState.mode`, I want us to start thinking about how we organize this. Maybe, what we should do is extending `GraphManagerState` and have `LegendStudioGraphManagerState` or something and put all graph handling logic of `EditorGraphState` in there, but colocating `GraphEditorMode` with `EditorGraphState` is sensible as it is what we had before